harness/determined/cli/trial.py

```diff
@@ -130,6 +131,7 @@ def get_with_offset(offset: int) -> bindings.v1GetTrialWorkloadsResponse:
             trial.experimentId,
             trial.state,
             json.dumps(trial.hparams, indent=4),
+            json.dumps(trial.summaryMetrics, indent=4),
```

Comment:
Thoughts about the what to print when `summaryMetrics` are empty?

Reply:
as is, this will just print an empty dict {}, i think that's sane?